sql/sql_expression.cc

```diff
@@ -1740,6 +1747,46 @@ unimp(std::shared_ptr<SqlExpression> lhs,
     throw HttpReturnException(400, "unimplemented operator " + op);
 }
 
+std::vector<std::shared_ptr<SqlExpression> >
+SqlExpression::
+findAggregators() const
+{
+    std::vector<std::shared_ptr<SqlExpression> > output;
+    std::vector<std::shared_ptr<SqlExpression> > children = getChildren();
+
+    int index = 0;
+    while(index < children.size()) {
+        auto child = children[index];
+
+        bool foundAggregator = false;
+        if (child->getType() == "function") {
+            const FunctionCallWrapper * function = dynamic_cast<const FunctionCallWrapper *>(child.get());
```

Comment:
It was originally a wrapper around builtin function calls and user function calls back when the syntax and usage was more different. We could rename it.